model/model.go

```diff
@@ -41,9 +46,13 @@ const (
 
 	// FeeTypeAmount represent FeeType in amount
 	FeeTypeAmount FeeType = "amount"
+
+	// RequestIDContextKey contact that holds the RequestID context key for
+	RequestIDContextKey Key = "api_RequestIDContextKey"
```

Comment:
can you update the `README.md` to include that one can set this `api_RequestIDContextKey` as the key for the request-id?? And that it should be a valid uuid string.

Reply:
Okay boss